src/SpendPermissionManager.sol

```diff
@@ -160,7 +164,7 @@ contract SpendPermissionManager is EIP712 {
     /// @param spendPermission Details of the spend permission.
     event SpendPermissionApproved(bytes32 indexed hash, address indexed account, SpendPermission spendPermission);
 
-    /// @notice SpendPermission was revoked prematurely by account.
+    /// @notice SpendPermission was revoked by account.
```

Comment:
i took this out because technically you can revoke at any time even after the permission has ended